src/OrchardCore/OrchardCore/Environment/Shell/Builders/StartupBaseMock.cs

```diff
@@ -14,14 +14,31 @@ internal class StartupBaseMock : StartupBase
         private readonly MethodInfo _configureService;
         private readonly MethodInfo _configure;
 
+        private readonly int _order;
+        private readonly int _configureOrder;
+
+        /// <inheritdoc />
+        public override int Order => _order;
```

Comment:
No need for the private field

```csharp
public override int Order { get; }
```

Reply:
Yes because here it's a special case, we are overriding properties but initialized from another instance by reflexion, and if the `ConfigureOrder` is not provided it defaults to the `Order` property ;) But you're right, i did it too quickly, i will make it simpler, thanks.